db/seeds.rb

```diff
@@ -11,11 +9,787 @@
+
+merchant = Merchant.new(id: '1001')
+merchant.email = 'woonyong92@gmail.com'
+merchant.password = '12345678'
+merchant.password_confirmation = '12345678'
+merchant.total_points = 0
+merchant.save
+
+merchant = Merchant.new(id: '1002')
+merchant.email = 'jkcheong92@gmail.com'
+merchant.password = '12345678'
+merchant.password_confirmation = '12345678'
+merchant.total_points = 0
+merchant.save
+
+merchant = Merchant.new(id: '1003')
+merchant.email = 'junwen29@gmail.com'
+merchant.password = '12345678'
+merchant.password_confirmation = '12345678'
+merchant.total_points = 0
+merchant.save
+
+merchant_id = 1004
+while merchant_id < 1100
+  merchant = Merchant.new(id: merchant_id)
+  merchant.email = merchant_id.to_s + '@gmail.com'
+  merchant.password = '12345678'
+  merchant.password_confirmation = '12345678'
+  merchant.total_points = 0
+  merchant.save
+  merchant_id = merchant_id + 1
+end
+
+# Seed payments and associated tables
+start_date = '01-12-2014'.to_date
+end_date = start_date + 1.months
+payment_id = 1003
+merchant_id = 1004
+plan_payment_id = 1003
+add_on_payment_id = 1004
+while merchant_id < 1010
+  Payment.new(id: payment_id, start_date: start_date, expiry_date: end_date, total_cost: '50.00', add_on1: false, add_on2: false, add_on3: false, plan1: false, paid: true, created_at: start_date, updated_at: '2015-10-01 00:00:00', merchant_id: merchant_id, months: 1).save(validate: false)
+  PlanPayment.create(id: plan_payment_id, plan_id: 1000, payment_id: payment_id, created_at: start_date, updated_at: '2015-10-01 00:00:00')
+  AddOnPayment.create(id: add_on_payment_id, add_on_id: 1000, payment_id: payment_id, created_at: start_date, updated_at: '2015-10-01 00:00:00')
+  add_on_payment_id = add_on_payment_id + 1
+  AddOnPayment.create(id: add_on_payment_id, add_on_id: 1001, payment_id: payment_id, created_at: start_date, updated_at: '2015-10-01 00:00:00')
+  start_date = end_date
+  end_date = end_date + 1.months
+  payment_id = payment_id + 1
+  merchant_id = merchant_id + 1
+  plan_payment_id = plan_payment_id + 1
+  add_on_payment_id = add_on_payment_id + 1
+end
+
+start_date = '01-12-2014'.to_date
+end_date = start_date + 1.months
+while merchant_id < 1050
+  Payment.new(id: payment_id, start_date: start_date, expiry_date: end_date, total_cost: '40.00', add_on1: true, add_on2: true, add_on3: false, plan1: true, paid: true, created_at: start_date, updated_at: '2015-10-01 00:00:00', merchant_id: merchant_id, months: 1).save(validate: false)
+  PlanPayment.create(id: plan_payment_id, plan_id: 1, payment_id: payment_id, created_at: start_date, updated_at: '2015-10-01 00:00:00')
+  AddOnPayment.create(id: add_on_payment_id, add_on_id: 1, payment_id: payment_id, created_at: start_date, updated_at: '2015-10-01 00:00:00')
+  add_on_payment_id = add_on_payment_id + 1
+  AddOnPayment.create(id: add_on_payment_id, add_on_id: 2, payment_id: payment_id, created_at: start_date, updated_at: '2015-10-01 00:00:00')
+  if merchant_id%rand(5..10) == 0
```

Comment:
this is for changing to the next month. If I do not put rand then the number of subscribers will be fixed each month making the line chart in payment analytics a straight line